port_drayage_plugin/include/port_drayage_plugin/port_drayage_worker.h

```diff
@@ -64,6 +65,7 @@ namespace port_drayage_plugin
             std::string _cmv_id;
             std::string _cargo_id;
             std::function<void(cav_msgs::MobilityOperation)> _publish_mobility_operation;
+            std::function<bool(cav_srvs::SetActiveRoute)> _call_set_active_route_client;
```

Comment:
It's a little bit confusing basically having a class member and a class method with the same name here, can one of them be adjusted? Is the public function strictly necessary or can that be made private or omitted possibly?

Reply:
I can't leave comments on line 73 below, but I believe the plugin name is supposed to obey legal ROS topic naming restrictions since it will be used to compose the actual ROS topic, just something I noticed.

Reply:
Just to clarify, for your first comment are you referring to the class member on Line 68 `std::function<bool(cav_srvs::SetActiveRoute)> _call_set_active_route_client;` and the public function on line 138 `cav_srvs::SetActiveRoute compose_set_active_route_request() const;`?

Reply:
I'm referring to that std::function `_call_set_active_route_client` and the `bool call_set_active_route_client(cav_srvs::SetActiveRoute req);` method on the same class.

Reply:
Thanks for the clarification! I adjusted the name in PortDrayageWorker and made the function private in PortDrayagePlugin.